docker/.env

```
@@ -542,5 +542,35 @@ SMQ_GRAFANA_ADMIN_PASSWORD=supermq
 ## Allow unverified user to access
 SMQ_ALLOW_UNVERIFIED_USER=true
 
+### NOTIFICATIONS SERVICE
+SMQ_NOTIFICATIONS_LOG_LEVEL=info
+SMQ_NOTIFICATIONS_HTTP_HOST=notifications
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_NOTIFICATIONS_HTTP_HOST key should go before the SMQ_NOTIFICATIONS_LOG_LEVEL key

(UnorderedKey)